kotlinx.fuzz.jazzer/CasrAdapter/src/lib.rs

```diff
@@ -0,0 +1,105 @@
+#![no_main]
```

Comment:
Have we tried looking into this implementation to see what it does? Can we implement something similar in Java or Kotlin in the future? Because we need to be sure that constantly dealing with a native library is worth the effort

Reply:
Also, have you thought about how this will work during publishing?

Reply:
As far as I see, it can be implemented in kotlin, but since we already have some working library I decided not to reinvent a bicycle.